components/RichBody/index.tsx

```diff
@@ -123,8 +135,10 @@ export default function RichBody(props: Props): ReactElement {
             <p
               dir={
                 typeof children[0] === 'object'
-                  ? getTextDirection(children[0])
-                  : direction(children)
+                  ? getTextDirectionFromNode(children[0])
```

Comment:
Is this for that case where you have empty strings on both sides of a node?

Reply:
that's right!